example/src/main/kotlin/dev/pbkit/wrp/example/MainActivity.kt

```diff
@@ -54,175 +34,45 @@ class MainActivity : ComponentActivity() {
 @Composable
 fun Test() {
     val scope = rememberCoroutineScope()
-    val sliderValue = remember { mutableStateOf(50f) }
-    val sliderValueFlow = remember { MutableSharedFlow<Float>() }
+    val slider = remember { MutableStateFlow(50f) }
     val inputText = remember { mutableStateOf(TextFieldValue("Hello, World!")) }
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
         Text(text = "WrpExampleServer (Host)")
         Slider(
-            value = sliderValue.value,
+            value = slider.collectAsState().value,
             valueRange = 0f..100f,
             onValueChange = {
-                scope.launch {
-                    sliderValue.value = it
-                    sliderValueFlow.emit(it)
-                }
+                slider.value = it
             }
         )
         TextField(
             value = inputText.value,
             onValueChange = { inputText.value = it }
         )
         WrpWebView(
+            scope = scope,
             update = { webView ->
                 WebView.setWebContentsDebuggingEnabled(true)
                 webView.loadUrl("https://pbkit.dev/wrp-example-guest")
             }
-        ) { event ->
-            val url = event.url
+        ) { _, socket, url ->
             Log.d("Wrp", "Socket is ready: $url")
             startWrpServer(
-                scope = event.webViewScope,
-                channel = WrpChannel(event.socket),
+                WrpChannel(socket),
                 serveWrpExampleService(object : WrpExampleService {
                     override suspend fun GetSliderValue(req: GetSliderValueRequest): ReceiveChannel<GetSliderValueResponse> {
                         val channel = Channel<GetSliderValueResponse>(Channel.BUFFERED)
```

Comment:
```suggestion
                        val channel = Channel<GetSliderValueResponse>()
```

`Channel.BUFFERED` 이거 제가 트러블슈팅 하면서 혹시나 하고 넣었던 건데, 이제 없어도 될거 같아요.